xpk.py

```diff
@@ -5833,6 +5890,37 @@ def workload_create(args) -> None:
         f' {args.workload} from the JobName filter on the dashboard.'
     )
 
+  if args.use_pathways:
```

Comment:
Have you considering have a Cloud Console UI page for pathways that could show these links. Then XPK only needs to direct to one link for example.